pkg/controller/defaults/placements.go

```diff
@@ -213,5 +213,41 @@ var (
 				},
 			},
 		},
+
+		"rgw-multi-instances": rook.Placement{
```

Comment:
Why are you creating a new entry? Can't you just modify the `rgw` one?

Reply:
The `rgw` entry can still be reused on a single `instance` mode. As you have pointed out we don't require `OBC` StorageClass in all scenarios, we may be switching between the (rgw / rgw-multi-instances) `Placement`s...

Reply:
Thanks for the clarification @jarrpa . Changed the existing `rgw` topology map itself.